README.md

```diff
@@ -553,3 +553,32 @@ tox must be configured to inherit these packages in the test environment. This c
 [vsc-ci]
 inherit_site_packages=1
 ```
+
+Pre-installing dependencies before running tests
+------------------------------------------------
+
+Although ``vsc-install`` will automatically install all dependencies listed in ``setup.py`` prior to running the
+tests, there are cases where this doesn't work out as expected.
+Some Python packages only support being installed with ``pip install`` (for example because they use a namespace
+that is spread across multiple different Python packages, like ``fs`` and ``fs.sshfs``).
+
+You can specify Python packages that should be installed (with ``pip install``) before running the tests via
```

Comment:
So this is only for tests?

Reply:
Yes. It's needed because `vsc-install` automagically installs the dependencies before running the tests, but the way that's done (via setuptools, not `pip`) is not compatible with some Python packages (like `fs.sshfs`).

For an actual installation, `fs.sshfs` can be installed some other way (not via `vsc-install`).